main.go

```diff
@@ -70,13 +82,35 @@ func doMain(args []string) {
 				Usage:  "Re-encrypt the secrets using the (new) encryption key",
 				Action: cmdReencrypt,
 			},
+			{
+				Name:   "sync-devices",
+				Usage:  "Synchronize device state across IoT platforms.",
+				Action: cmdSync,
+				Flags: []cli.Flag{
+					&cli.IntFlag{
+						Name:  "batch-size",
+						Usage: "Maximum number of devices to sync in a batch.",
+						Value: 50,
```

Comment:
Can we set a min and max value for this parameter? What is going to happen if I set an arbitrarily large number?
Or a negative number?

Reply:
There is a check down in `cmdSync` forcing batch size to be between 1 and 500.